docs/hypervisors.md

```diff
@@ -0,0 +1,48 @@
+# Hypervisors
+
+* [Hypervisors](#hypervisors)
+    * [Types](#types)
+    * [Determine currently configured hypervisor](#determine-currently-configured-hypervisor)
+    * [Choose a Hypervisor](#choose-a-hypervisor)
+
+## Types
+
+Kata Containers supports multiple hypervisors. Since they offer different
+features and options, Kata Containers provides a separate
+[configuration file](/src/runtime/README.md#configuration)
+for each.
+
+> **Note:**
+>
+> The simplest way to switch between hypervisors is to create a symbolic link
+> to the appropriate hypervisor-specific configuration file.
+
+| Hypervisor | Written in | Architectures | Type | Comment | Configuration file |
+|-|-|-|-|-|-|
+[ACRN](https://projectacrn.org/) | C | `x86_64` | Type 1 | "bare metal" | `configuration-acrn.toml` |
+[Cloud Hypervisor](https://github.com/cloud-hypervisor/cloud-hypervisor) | rust | `aarch64`, `x86_64` | Type 2 | [KVM] | `configuration-clh.toml` |
+[Firecracker](https://github.com/firecracker-microvm/firecracker) | rust | `aarch64`, `x86_64` | Type 2 | [KVM] | `configuration-fc.toml` |
+[QEMU](http://www.qemu-project.org/) | C | all | Type 2 | [KVM] | `configuration-qemu.toml` |
+
+[KVM]: https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine
+
+## Determine currently configured hypervisor
```

Comment:
In so far as you can pass a configuration option, I would not call it the "configured hypervisor", maybe the "default hypervisor"?

Reply:
Well, it used to say "default", but folk complained that there "isn't a default" or "it depends on ( how it's packaged | who packaged it )" :smile: